src/game.cpp

```diff
@@ -2889,8 +2889,13 @@ void game::write_memorial_file( std::string sLastWords )
 
 void game::debug()
 {
+    std::string debugText = _( "Debug Functions - Using these is CHEATING!" );
+    if( one_in( 100 ) ) {
+        debugText =
+            _( "Using these will cheat not only the game, but yourself.  You won't grow. You won't improve.  Taking this shortcut will gain you nothing. Your victory will be hollow.  Nothing will be risked and nothing will be gained." );
```

Comment:
```suggestion
            _( "Using these will cheat not only the game, but yourself.  You won't grow.  You won't improve.\nTaking this shortcut will gain you nothing.  Your victory will be hollow.\nNothing will be risked and nothing will be gained." );
```

Reply:
How it looks with line breaks:
![line-breaks](https://user-images.githubusercontent.com/1931904/56098122-34b57980-5f0e-11e9-9d87-cc19f9f864eb.png)